README.md

```diff
@@ -17,6 +17,7 @@ Table of contents -
  * [Installation](#installation)
  * [Initialization](#initialization)
```

Comment:
Should we mention the solution suggested in below issue in Initialization section?
https://github.com/imagekit-developer/imagekit-java/issues/10